### DISCUSSION TITLE
Make error messages in UI selectable

### DISCUSSION THREAD

**Discussion by sagerio** [directus/directus#4553] [Feature Requests]
It would be comfortable if the error messages in red where selectable so one can copy them instead of typing it again.

![image](https://user-images.githubusercontent.com/29227014/111292009-08e62580-8648-11eb-9b2e-acf8732139d0.png)

**Comment by benhaynes** [comment]
To add on to this, I think we should have a dedicated "copy" button.

**Reply by sagerio** [reply]
Would be even better 👍🏻

**Reply by benhaynes** [reply]
@rijkvanzanten I think this warrants an issue... at least to get the text selectable. No "convert to issue" action it looks like.

**Reply by benhaynes** [reply]
Ref: https://github.com/directus/directus/issues/4554

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
